<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>convWindow</class>
 <widget class="QDockWidget" name="convWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>200</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>300</width>
    <height>200</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Gerar azimute e distância</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="pointsButton">
        <property name="text">
         <string>Selecionar geometria ou pontos</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QRadioButton" name="decimalButton">
        <property name="text">
         <string>Graus Decimais</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="dmsButton">
        <property name="text">
         <string>Grau minuto segundo</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTreeWidget" name="mapList">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::SingleSelection</enum>
      </property>
      <attribute name="headerCascadingSectionResizes">
       <bool>false</bool>
      </attribute>
      <attribute name="headerStretchLastSection">
       <bool>false</bool>
      </attribute>
      <column>
       <property name="text">
        <string>Ponto</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>X</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Y</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Azimute</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Distância</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Destino</string>
       </property>
      </column>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="csvButton">
      <property name="text">
       <string>Exportar como CSV</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
